release.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+CURRENT_DIR=$(pwd)
+MERGE_COMMIT_MESSAGE="MEP $(date +'%d.%m.%Y') : Merge branch 'main' into release"
+GIT_BACK_END_URL=git@github.com:gip-inclusion/dora-back.git
+GIT_FRONT_END_URL=git@github.com:gip-inclusion/dora-front.git
+
+TEMP_DIR=$(mktemp -d)
+echo "Création d'un répertoire temporaire pour le travail : $TEMP_DIR"
+cd $TEMP_DIR
```

Comment:
```suggestion
cd "$TEMP_DIR"
```

Pas strictement nécessaire, parce que mktemp ne va jamais insérer d’espace, mais par  cohérence avec les remarques suivantes

Reply:
yes ! bien vu. je prends.